day3-b.js

```diff
@@ -0,0 +1,33 @@
+//**Hello Ramin this is part 1 of day 3 :D **//
+//i am running this online to get the input here : https://www.codechef.com/ide //
+process.stdin.resume();
+process.stdin.setEncoding('utf8');
+
+
+process.stdin.on('data', function(data) { 
+    
+    var test = data.split("\n")
+		var zero = 0 ;
+	var one = 0 ;
+	var gamma= "";
+	var epsilon =""
+		let i = 0
+	 while ( i < test[1].length )  {
+		for(let j=0 ; j<test.length ; j++)	{		
```

Comment:
Why do you use first `while` and then `for`? This makes me as a reviewer wonder if you want to achieve different things, but you actually aren't.

```suggestion
	 for (let i = 0; i < test[1].length; i++)  {
		for (let j = 0; j < test.length; j++)	{		
```